src/blocks/common.blocks/block-example/block-example.js

```diff
@@ -1,6 +1,6 @@
-modules.define('i-bem__dom', ['jquery'], function (provide, $, BEMDOM) {
+modules.define('block-example', ['i-bem__dom', 'jquery', 'dropdown'], function (provide, BEMDOM, $, Dropdown) {
 
-BEMDOM.decl('block-example', {
+BEMDOM.decl(this.name, {
```

Comment:
In this case you `block-example` module will provide original `BEMDOM` class, not `BEMDOM.blocks['block-example']`